app/client/ts/video-model.ts

```diff
@@ -25,7 +25,7 @@ class UnsupportedVideoFormatError extends Error {
     }
 }
 
-export default class Video implements Drop {
+export class Video implements Drop {
```

Comment:
Video really is the main thing this module exports. You can have both a default export and standard exports.